.github/script/get-prev-version

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
```

Comment:
Can we use JS instead of ruby 😅 ? Will help us remove ruby step if we can.

Reply:
Well, we try to not use any ruby script in our build process. This is just for GitHub action so I don't see the need :D